test/base/__init__.py

```diff
@@ -100,3 +113,114 @@ def forNode(self):
             'expires': self.expires,
             'scopes': self.scopes
         }
+
+
+class FakeGenerated(object):
```

Comment:
Multi inheritance doesn't feel like the right approach here.  Could we instead have a method like
```python
def createFakeApi(cls, *args, **kwargs) {
  instance = cls(*args, **kwargs)
  instance._makeHttpRequest = mock.MagicMock()
  instance._makeTopicExchange = mock.MagicMock()
}
```
?